eth2/operation_pool/src/lib.rs

```diff
@@ -420,483 +351,344 @@ fn prune_validator_hash_map<T, F, E: EthSpec>(
 impl<T: EthSpec + Default> PartialEq for OperationPool<T> {
     fn eq(&self, other: &Self) -> bool {
         *self.attestations.read() == *other.attestations.read()
-            && *self.deposits.read() == *other.deposits.read()
             && *self.attester_slashings.read() == *other.attester_slashings.read()
             && *self.proposer_slashings.read() == *other.proposer_slashings.read()
             && *self.voluntary_exits.read() == *other.voluntary_exits.read()
     }
 }
 
-#[cfg(test)]
-mod tests {
-    use super::DepositInsertStatus::*;
+// TODO: more tests
+#[cfg(not(debug_assertions))]
```

Comment:
Should be `#[cfg(all(test, not(debug_assertions)))]` to avoid a compiler warning when not running the tests (see https://github.com/pscott/lighthouse/runs/345700007#step:5:757)

Reply:
good catch!